Great!

Could You please fix:

```
xwid@xwid:~/Projekty/forked/lg-webos-client$ cargo test
error[E0308]: mismatched types
 --> examples/get-channel-list/main.rs:7:39
  |
7 |     let mut client = WebosClient::new("ws://192.168.1.62:3000/").await.unwrap();
  |                                       ^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `WebOsClientConfig`, found `&str`

error: aborting due to previous error
```

And then I'll merge.

--- a/examples/get-channel-list/main.rs
+++ b/examples/get-channel-list/main.rs
@@ -1,10 +1,12 @@
-use lg_webos_client::client::WebosClient;
+use lg_webos_client::client::*;
 use lg_webos_client::command::Command;
 
 #[tokio::main]
 async fn main() {
     env_logger::init();
-    let mut client = WebosClient::new("ws://192.168.1.62:3000/").await.unwrap();
+    // Note: We must specify the ws protocol, and if we do not have the key, we must just use a blank str.
+    let config = WebOsClientConfig::new("ws://192.168.1.62:3000/", "");
+    let mut client = WebosClient::new(config).await.unwrap();
     let resp = client.send_command(Command::GetChannelList).await.unwrap();
     println!("Got response {:?}", resp.payload);
 }

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -10,7 +10,7 @@
 //! #[tokio::main]
 //! async fn main() {
 //!     env_logger::init();
-//!     // Note: We must specify the ws protocol, and if we do not have the key, we must just use a blank str.
+//!     // Note: We must specify the ws protocol, and if we do not have the key, we must use a blank str.
 //!     let config = WebOsClientConfig::new("ws://192.168.1.62:3000/","");
 //!
 //!     let mut client = WebosClient::new(config).await.unwrap();

